scripts/validate-upgrade-ssv-network-views.ts

```diff
@@ -0,0 +1,14 @@
+async function main() {
+  const proxyAddress = process.env.SSVNETWORKVIEWS_PROXY_ADDRESS;
+  const SSVNetworkViews = await ethers.getContractFactory("SSVNetworkViews_V2");
+
+  await upgrades.validateUpgrade(proxyAddress, SSVNetworkViews, { kind: 'uups' });
+  console.log("Contract validation finished");
+}
+
+main()
```

Comment:
maybe to change main to something more specific like you did with scripts/upgrade-ssv-network.ts